src/Payum/LaravelPackage/Controller/NotifyController.php

```diff
@@ -15,8 +16,12 @@ public function doUnsafeAction(Request $request)
         return \Response::make(null, 204);
     }
 
-    public function doAction(Request $request)
+    public function doAction($payumToken)
     {
+        /** @var Request $request */
+        $request = app('request');
+        $request->attributes->set('payum_token', $payumToken);
```

Comment:
The way the route is registered in the PayumServiceProvider creates a uri like `http://my-page.com/payment/notify/token-here/`
```php
$route->get('/payment/notify/{payum_token}', array(
    'as' => 'payum_notify_do',
    'uses' => 'Payum\LaravelPackage\Controller\NotifyController@doAction'
));
```
Laravel then passes the token to the controller method instead of including it in the Request as before. The way it was before made payum throw an exception because $request didn't contain payum_token as required.